pom.xml

```diff
@@ -230,6 +230,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.12.1</version>
```

Comment:
Why did we need to put versions of the plugins in here? I believe we strive to manage those in the `root-parent-pom`

Reply:
Yep - we should not be adding these